google-cloud-pubsub/samples/acceptance/schemas_test.rb

```diff
@@ -75,13 +78,19 @@
     let(:record) { { "name" => "Alaska", "post_abbr" => "AK" } }
 
     it "supports pubsub_create_topic_with_schema, pubsub_publish_avro_records with binary encoding" do
-      @schema = pubsub.create_schema schema_id, :avro, avsc_definition
+      schema = Google::Cloud::PubSub::V1::Schema.new name: schema_id, 
+                                                     type: :AVRO,
+                                                     definition: avsc_definition
+      @schema = schemas.create_schema parent: pubsub.project_path,
+                                      schema: schema,
+                                      schema_id: schema_id
+
 
       # pubsub_create_topic_with_schema
       assert_output "Topic projects/#{pubsub.project}/topics/#{topic_id} created.\n" do
-        create_topic_with_schema topic_id: topic_id, schema_id: schema_id, message_encoding: :binary
+        create_topic_with_schema topic_id: topic_id, schema_id: schema_id, message_encoding: "BINARY"
```

Comment:
I'm noticing that some enum values are now being passed in as strings (e.g. `"BINARY"` here) while others are being passed in as symbols (e.g. `:AVRO` a few lines up). That's likely to be confusing to users who might wonder when to use one vs the other. If these are enum values, then protobuf should be able to take either strings or symbols, so can we standardize on one or the other? (I think I'd prefer symbols.)